### PR TITLE
docs(readme): update to better reflect design and philosophy of Combo

### LINKED ISSUE #5: update combo README
# Story
As a developer trying to use Combo,
I want a README that better describes the repository
So that I can get standard questions answered faster.

# Summary
The current README could do with an overall that better answers the commonly answered questions around Combo.

# Acceptance Criteria

- Description of why Combo exists
- Example of using on/off cluster
- Alternatives and why you may not want to use them
- Answers to commonly asked questions

### PR DESCRIPTION
**Pull request by tylerslaton**
# Summary
Updating the README.md to better reflect a comment that @njhale left on #4 which outlined a much better structure for it. On top of his suggestion, I added some changes to tie it all together.

# Additions
- Description of what Combo is
- Example of using on/off cluster
- Alternatives and why you may not want to use them

Closes #5

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -224,3 +232,11 @@ status:
...
       ...
 ```
+
+## Ulterior motives
+
+Our "hidden" agenda with `combo` is for it to:
+- serve as an example of best-practices when building operators
+- be used to dog food operator-framework packaging formats (e.g. OLM, rukpak, etc)
+- become a dependency of operators that require post-install configuration (e.g. RBAC scoping, secret creation, etc)
+- serve as a kruft-free way to up-skill new maintainers through easy contributions
```
Comment (njhale) [inline README.md:242]: nit: Let's use the common spelling of "cruft"

**Review by njhale — CHANGES_REQUESTED** [review]
Looking good!

@tylerslaton what do you think about adding a checklist of planned/implemented features?

File: README.md (hunk 1 above)
Comment (tylerslaton) [reply README.md:242]: Good call 😅

**Comment by tylerslaton** [comment]
> Looking good!
> 
> @tylerslaton what do you think about adding a checklist of planned/implemented features?

I like that idea! Let me draft out what that looks like and put it up with this PR.

**Pull request merged**